datastore/csv.go

```diff
@@ -0,0 +1,46 @@
+package datastore
+
+import (
+	"encoding/csv"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/omarscd/academy-go-q42021/model"
+)
+
+func NewSUSMap() map[uint64]model.StandUser {
```

Comment:
By convention, every exported stuff (function, variable, constant, etc) should have a description

```go
// NewSUSMap - Description here....
func NewSUSMap() map[uint64]model.StandUser {
```